content/app/development/configuration/stateless/_index.en.md

```diff
@@ -438,31 +435,31 @@ Videre i eksempelet vil betegnelsen *bruker* være synonymt med en virksomhet re
+In a stateless application you have the opportunity to use the `InstantiationButton` component to start an instance.
+Currently we only support starting an instance from within the same application the stateless form is displayed in. Starting an instance in another application is functionality we will add in the future.
 
-Det er laget en eksempel applikasjon som er satt opp som en innsynstjeneste hvor sluttbruker kan velge å starte en instans på den aktuelle applikasjonen. Denne kan brukes til inspirasjon for videre utvikling. Applikasjonen med kildekode finnes [her.](https://altinn.studio/repos/ttd/start-from-stateless)
+There is an example application which is set up like an access service where the end user can choose to start an instance in the actual application. It can be used as inspiration for further development. The application with source code can be found [here](https://altinn.studio/repos/ttd/start-from-stateless)
 
-### Instansiere med prefill
+### Instantiation with prefill
 
-Et bruksområde for det å starte en instans fra et stateless view kan være at man først ønsker at appen skal oppføre seg som en innsynstjeneste hvor brukeren blir presentert for gitte data som er aktuell. Fra denne informasjonen kan brukeren velge å agere videre på dataen som listes opp, og da er man over på en vanlig innsendingstjeneste.
+One use case where starting an instance from a stateless view can be that you first want the app to act like an access service where the user is presented some given data. From this information the user can choose to act on the listed data, and then use the service as a normal form submittance service.
 
-For å få til en slik flyt er man først avhengig av å sette opp applikasjonen som en stateless appliksjon som står beskrevet under [konfigurasjon.](#konfigurasjon)
-Når dette er gjort kan man utvide stateless viewet til å inkludere `InstantiationButton` som kan vil starte en ny instans i det brukeren klikker på knappen.
-Standard oppførsel for denne knappen er å sende inn hele datamodellen som brukeren har benyttet inn som en del av instansieringn under feltet `prefill`. 
-Om man ønsker å velge ut deler av datamodellen som er benyttet i det tilstandsløse steget vil det også være mulig ved å legge til `mapping` på `InstantiationButton`-komponenten. F.eks
+To achieve this type of flow you are dependent on setting up your application as a stateless application, described under [configuration](#Configuration).
+When this has been done, you can expand the stateless view to include `InstantiationButton` which can start a new instance when the user clicks the button.
+Standard behaviour for this button is to submit the entire datamodel that the user has used as a part of the instantiation under the field `prefill`.
+If one wishes to pick parts of the datamodel which are connected in the stateless step it will also be possible to add `mapping`to the `InstantiationButton` component. For example:
```

Comment:
```suggestion
Standard behaviour for this button is to submit the entire data model that the user has used as a part of the instantiation under the field `prefill`.
If one wishes to pick parts of the data model which are connected in the stateless step it will also be possible to add `mapping`to the `InstantiationButton` component. For example:
```